fuzzamoto-ir/src/generators/block.rs

```diff
@@ -36,8 +36,13 @@ impl<R: RngCore> Generator<R> for BlockGenerator {
         let block_version_var =
             builder.force_append_expect_output(vec![], Operation::LoadBlockVersion(5));
 
+        let coinbase_tx_var = builder
+            .get_random_variable(rng, Variable::CoinbaseTx)
+            .ok_or(GeneratorError::MissingVariables)?;
```

Comment:
Could we generate a coinbase if it's missing here instead?

Reply:
I changed `BlockGenerator` to contain `CoinbaseTxGenerator` and removed it from the generator list since `CoinbaseTx` is only consumed by `BuildBlock`. I could also merge it with `BlockGenerator` so it lives in `generators/block.rs` instead of having it be a sub-generator, but currently it needs access to helper functions in `generators/tx.rs`.